app/src/App.tsx

```diff
@@ -173,52 +176,54 @@ function App() {
   useCompile(codeToCompile, setError, setIsCompiled, updateLog, toolchain);
 
   return (
-    <div
-      style={{
-        padding: '15px',
-        margin: '0px',
-        background: '#F1F1F1',
-      }}>
-      <ActionToolbar
-        deployState={deployState}
-        setContractId={setContractId}
-        onShareClick={onShareClick}
-        onCompile={onCompileClick}
-        isCompiled={isCompiled}
-        setDeployState={setDeployState}
-        drawerOpen={drawerOpen}
-        setDrawerOpen={setDrawerOpen}
-        showSolidity={showSolidity}
-        setShowSolidity={setShowSolidity}
-        updateLog={updateLog}
-      />
+    <ThemeContext.Provider value={{setTheme,theme}}>
```

Comment:
Rather than passing in the value, we can make the Provider check the theme context.